search/search.go

```diff
@@ -43,25 +45,18 @@ func Search(fs afero.Fs, scope, query string, checker rules.Checker, found func(
 		}
 
 		if len(search.Conditions) > 0 {
-			match := false
-
 			for _, t := range search.Conditions {
 				if t(path) {
-					match = true
-					break
+					return found(originalPath, f)
```

Comment:
This is not correct. If user entered a search query the result will not be filtered.